### PR TITLE
exporter: add field alias for count(*) func

### PR DESCRIPTION
**Pull request by liubin**
Thank you for contributing to Harbor!

# Comprehensive Summary of your change

Add an alias so that orm can parse the column name regardless of
    the type and version of the database.

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [x] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by wy65701436 — APPROVED** [review]
lgtm

**Comment by chlins** [comment]
@liubin Hi, the output of the original and new SQL is same in my environment, you mentioned that the original SQL will always got 0 count, how did you reproduce that?

image

**Comment by liubin** [comment]
@chlins  Sorry I'm using oceanbase, it needs alias to let the beego orm to parse the filed name correctly.

I updated the commit message and description of this PR.

**Review by chlins — APPROVED** [review]
lgtm

File: src/pkg/exporter/project_collector.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 const ProjectCollectorName = "ProjectCollector"
 
 var (
-	totalProjectSQL = `SELECT project_metadata.value AS public, COUNT(project_metadata.value)
+	totalProjectSQL = `SELECT project_metadata.value AS public, COUNT(project_metadata.value) as count
```
Comment (chlins) [inline src/pkg/exporter/project_collector.go:31]: Please use uppercase `AS` for consistency.

Comment (liubin) [reply src/pkg/exporter/project_collector.go]: @chlins updated.

**Pull request merged**